<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionQuit"/>
    <addaction name="actionSaveSession"/>
    <addaction name="actionRecent"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomFit"/>
   <addaction name="actionPanLeft"/>
   <addaction name="actionPanRight"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="timeSpanDockWidget">
   <property name="windowTitle">
    <string>Timespan selection</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="signalsDockWidget">
   <property name="windowTitle">
    <string>Signal sources</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionZoomFit">
   <property name="text">
    <string>Zoom fit</string>
   </property>
   <property name="toolTip">
    <string>Autoscale</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionPanRight">
   <property name="text">
    <string>Pan Right</string>
   </property>
  </action>
  <action name="actionPanLeft">
   <property name="text">
    <string>Pan Left</string>
   </property>
  </action>
  <action name="actionSaveSession">
   <property name="text">
    <string>Save session</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent session...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
